cypress/integration/pages/onDemandRadioEpisode/helpers.js

```diff
@@ -0,0 +1,32 @@
+import envConfig from '../../../support/config/envs';
+
+// the externalId `bbc_oromo_radio` is overriden to `bbc_afaanoromoo` in production code
+const getBrandId = externalId => {
+  return externalId === 'bbc_oromo_radio'
+    ? 'bbc_afaanoromoo_radio'
+    : externalId;
+};
+
+const getServiceName = producerName => {
+  const producerNameLowerCase = producerName.toLowerCase();
+  return producerNameLowerCase === 'indonesian'
+    ? 'indonesia'
+    : producerNameLowerCase;
+};
```

Comment:
Could be simplified to this 🙂 
```suggestion
const getServiceName = producerName =>
  producerName.toLowerCase().replace('indonesian', 'indonesia');
```

Reply:
Changed